src/SwedbankPay.Sdk/Models/Callbacks/Callback.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace SwedbankPay.Client.Models.Callbacks
+﻿namespace SwedbankPay.Sdk.Models.Callbacks
```

Comment:
I don't think `Models` provide any useful information in a namespace. Everything that is a `class`, `struct` or `interface` is a model. Please remove it.